packages/react-composites/src/composites/CallComposite/Strings.tsx

```diff
@@ -173,4 +173,49 @@ export interface CallCompositeStrings {
    * Tooltip text used to inform a user that toggling microphone in lobby is not supported.
    */
   microphoneToggleInLobbyNotAllowed: string;
+  /* @conditional-compile-remove(call-composite-participant-pane) */
+  /**
+   * Side pane People section Title.
+   */
+  peoplePaneTitle: string;
+  /* @conditional-compile-remove(call-composite-participant-pane) */
+  /**
+   * Aria label string for return to call back button
+   */
+  returnToCallButtonAriaLabel?: string;
+  /* @conditional-compile-remove(call-composite-participant-pane) */
+  /**
+   * Aria Description string for return to call button
+   */
+  returnToCallButtonAriaDescription?: string;
+  /* @conditional-compile-remove(call-composite-participant-pane) */
+  /**
+   * control bar People button label
+   */
+  peopleButtonLabel: string;
+  /* @conditional-compile-remove(call-composite-participant-pane) */
+  /**
+   * control bar Chat button label.
+   */
+  chatButtonLabel: string;
```

Comment:
chatButtonlabel is used in TabHeader.tsx which was made into a common file. This could be resolved if we have a common set of strings, otherwise, this will be needed in CallCompositeStrings as well. 

For now, as this is in beta, I have placed it in CallCompositeStrings. This will need to be discussed further at a later date.